pkg/controller_test/zz_plumbing_for_test.go

```diff
@@ -148,13 +163,49 @@ func (tc *testCase) run(t *testing.T) {
 		crdClient.AddReactor(reactor.verb, reactor.resource, reactor.reactor(t))
 	}
 	var scClient scClientset.Interface
+	var catalogger *catalog.Catalog
 	if tc.enableServiceCatalog {
-		scClientFake := scFake.NewSimpleClientset(tc.scClientObjects...)
+		scClientFake := scFake.NewSimpleClientset(append(
+			tc.scClientObjects,
+			[]runtime.Object{
+				&sc_v1b1.ClusterServiceClass{
+					TypeMeta: meta_v1.TypeMeta{
+						Kind:       "ClusterServiceClass",
+						APIVersion: sc_v1b1.SchemeGroupVersion.String(),
+					},
+					ObjectMeta: meta_v1.ObjectMeta{
+						Name: serviceClassName,
+					},
+					Spec: sc_v1b1.ClusterServiceClassSpec{
+						ExternalName: serviceClassExternalName,
+						ExternalID:   serviceClassName,
+					},
+				},
+				&sc_v1b1.ClusterServicePlan{
+					TypeMeta: meta_v1.TypeMeta{
+						Kind:       "ClusterServicePlan",
+						APIVersion: sc_v1b1.SchemeGroupVersion.String(),
+					},
+					ObjectMeta: meta_v1.ObjectMeta{
+						Name: servicePlanName,
+					},
+					Spec: sc_v1b1.ClusterServicePlanSpec{
+						ClusterServiceClassRef: sc_v1b1.ClusterObjectReference{
+							Name: serviceClassName,
+						},
+						ExternalName: servicePlanExternalName,
+						ExternalID:   servicePlanName,
+					},
+				},
+			}...)...,
```

Comment:
more dots

Reply:
yeah... this was a bit silly. I was a little bit too keen to write it as one expression, I guess.